syntax = "proto3";

import "tx_log.proto";

message log_entry {
  uint64 term = 1;
  uint64 index = 2;
  repeated tx_log xlog = 3;
}

message log_state {
  uint64 term = 1;
  uint32 vote = 2;
  uint64 commit_index = 3;
  uint64 consistency_index = 4;
}